run_oscar.json

```diff
@@ -0,0 +1,107 @@
+{
+    "id": "run_oscar",
+    "summary": "Run an external OSCAR service.",
+    "description": "Run an external docker-based OSCAR service.",
+    "categories": [
+        "oscar",
+        "service"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "token_env_var",
+            "description": "OpenEO auth token environment variable.",
+            "schema": {
+                "title": "token-env-var",
+                "type": "string",
+                "subtype": "string",
+                "pattern": "^[a-zA-Z0-9_\\-\\.]+$"
+            }
+        },
+        {
+            "name": "oscar_endpoint",
+            "description": "The URL of the OSCAR service.",
+            "schema": {
+                "title": "oscar-url",
+                "type": "string",
+                "format": "url",
+                "subtype": "url",
+                "pattern": "^https?://"
+            }
+        },
+        {
+            "name": "service",
+            "description": "The OSCAR service to be executed.",
+            "schema": {
+                "title": "service",
+                "type": "string",
+                "subtype": "string"
+            }
+        },
+        {
+            "name": "output",
+            "description": "The output path for the result.",
+            "schema": {
+                "title": "output",
+                "type": "string",
+                "subtype": "string"
+            }
+        },
+        {
+            "name": "input_file",
+            "description": "input file to be uploaded.",
+            "schema": {
+                "title": "input-file",
+                "type": "string",
+                "subtype": "string"
+            }
+        },
+        {
+            "name": "service_config",
```

Comment:
Do we pass this config file as inline JSON object or as an URL to a publicly available file?